Client/WebAuthN/U2FExtensions.swift

```diff
@@ -373,26 +431,49 @@ class U2FExtensions: NSObject {
             
             makeCredentialRequest.excludeList = exclusionList
             
-            guard let fido2Service = YubiKitManager.shared.keySession.fido2Service else {
-                self.sendFIDO2RegistrationError(handle: handle)
-                return
-            }
-            
-            fido2Service.execute(makeCredentialRequest) { [weak self] response, error in
+            executeKeyRequestWith { [weak self] in
                 guard let self = self else {
-                    log.error(U2FErrorMessages.ErrorRegistration.rawValue)
                     return
                 }
-                guard error == nil else {
-                    self.handleMakeCredential(handle: handle, request: request, error: error)
+                guard let fido2Service = self.keyFido2Service else {
+                    self.sendFIDO2RegistrationError(handle: handle)
                     return
                 }
                 
-                guard let response = response else {
-                    self.sendFIDO2RegistrationError(handle: handle)
-                    return
+                fido2Service.execute(makeCredentialRequest) { [weak self] response, error in
+                    guard let self = self else {
+                        log.error(U2FErrorMessages.ErrorRegistration.rawValue)
+                        return
+                    }
+                    guard error == nil else {
```

Comment:
Switch to `if`. We generally use `guard` only if an object needs to be unwrapped